modules/builds-using-image-change-triggers.adoc

```diff
@@ -87,11 +87,13 @@ imageChange:
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated[]
 In addition to setting the image field for all `Strategy` types, for custom builds, the `OPENSHIFT_CUSTOM_BUILD_BASE_IMAGE` environment variable is checked.
-If it does not exist, then it is created with the immutable image reference. If it does exist then it is updated with the immutable image reference.
+If it does not exist, then it is created with the immutable image reference. If it does exist, then it is updated with the immutable image reference.
 endif::[]
 
 If a build is triggered due to a webhook trigger or manual request, the build that is created uses the `<immutableid>` resolved from the `ImageStream` referenced by the `Strategy`. This ensures that builds are performed using consistent image tags for ease of reproduction.
 
+To get the ID of the last image that triggered an `imageChange` trigger, inspect the value of `lastTriggeredImageID` in the `BuildConfig` status.
```

Comment:
there can be more than on image change trigger, there is an array of triggers in the spec

for a given image change trigger, both the spec and status a `from` field with the `ImageStreamTagReference` .... you can use those to correlate which image stream tag a particular `lastTriggeredImageID` pertains to.